fusor-ember-cli/server/mocks/management-application.js

```diff
@@ -43,17 +259,5 @@ module.exports = function(app) {
     });
   });
 
-  managementApplicationsRouter.put('/:id', function(req, res) {
-    res.send({
-      'subscription': {
-        id: req.params.id
-      }
-    });
-  });
-
-  managementApplicationsRouter.delete('/:id', function(req, res) {
-    res.status(204).end();
-  });
-
-  app.use('/api/v21aaaaaa/management_applications', managementApplicationsRouter);
+  app.use('/customer_portal222/owners', managementApplicationsRouter);
```

Comment:
note, api mock is disabled as the path is `/customer_portal222/owners`. It would be `/customer_portal/owners` if enabled.